plugin/storage/cassandra/Dockerfile

```diff
@@ -0,0 +1,7 @@
+FROM jpkroehling/cassandra
```

Comment:
This was discussed on the mailing list. I'll either change this to use the "official" Dockerhub image for Cassandra or change to use an image we are building for OpenShift. I don't expect to change this before this PR is merged, though.